web/sidebars.js

```diff
@@ -148,10 +148,12 @@ module.exports = {
             'guides/auth/auth-apple',
             'guides/auth/auth-bitbucket',
             'guides/auth/auth-discord',
+            'guides/auth/auth-email',
```

Comment:
Let's move both of these (email, magic link) to the top of the list since they are the most common